pycbc/inference/likelihood.py

```diff
@@ -677,6 +679,9 @@ def __init__(self, waveform_generator, data, f_lower, psds=None,
         self.set_lognl(-0.5*sum([
             d[kmin:kmax].inner(d[kmin:kmax]).real
             for d in self._data.values()]))
+        # store the calibration transfer functions and params
+        self.tfs = transfer_functions
+        self.calib = calib
```

Comment:
Should just need only a ``self.calibration = calibration_instance``. Don't need a ``self.tfs``.